docs/releasenotes.md

```diff
@@ -5,6 +5,24 @@ title: Releasenotes
 
 # Release notes BAG Huidige bevragingen API
 
+## Versie 1.2.0 (22-06-2021)
+In deze release zijn enkele bugs in productie opgelost. Tevens zijn er enkele wijzigingen aan de API-specificaties en de feature-files aangebracht.
+Er worden nu adresregels geleverd met daarin het samengestelde adres.
+De Header Content-Crs is voorzien van een default waarde.
+Op /adressen worden de resultaten gesorteerd geretourneerd.  
+
+### Opgeloste issues
+
+  | Link Github	| Beschrijving |
+  | --- | --- |
+  | [338](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/375 ) | Underscores verwijderen uit componentnamen en enumeratienamen. |
+  | [375](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/375 ) | Als gebruiker wil ik gesorteerde resultaten in combinatie van pagineren |  
+  | [370](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/370) | Http status code 415 voor fout "Coördinatenstelsel {crs} in Content-Crs wordt niet ondersteund." incorrect |  
+  | [360](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/360) | Als gebruiker wil ik het samengesteld adres krijgen |
+  | [294](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/294) | Als gebruiker (nader te bepalen) wil ik de inhoud van de mogelijkOnjuist gegevensgroep kunnen filteren |
```

Comment:
is dat "(nader te bepalen)" nodig of mogen we dat hier en in het issue verwijderen?

Reply:
370 verwijderd uit de lijst opgeloste issues. Lijst van Frank (https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues?q=is%3Aissue+label%3Av1.2+is%3Aclosed) opgenomen in de releasenotes.